core/modules/admin_bar/admin_bar.inc

```diff
@@ -719,16 +719,22 @@ function admin_bar_theme_settings() {
     '#type' => 'checkboxes',
     '#title' => t('Enabled components'),
     '#options' => array(
-      'admin_bar.icon' => t('Icon menu'),
-      'admin_bar.menu' => t('Management menu'),
-      'admin_bar.search' => t('Search bar'),
-      'admin_bar.users' => t('User counts'),
-      'admin_bar.account' => t('Account links'),
+      'admin_bar.icon' => t('Home menu'),
+      'admin_bar.menu' => t('Administration menu'),
+      'admin_bar.search' => t('Menu search'),
+      'admin_bar.users' => t('Online'),
+      'admin_bar.account' => t('User account links'),
     ),
     '#default_value' => $config->get('components'),
     '#description' => t('These features will be enabled/visible in the administration bar. Untick the boxes to disable/hide them.'),
   );
 
+  $form['components']['admin_bar.icon']['#description'] = t('A specialized menu that links to the home page and provides links to perform various other admin tasks (e.g.: clear caches, run cron, etc.).');
+  $form['components']['admin_bar.menu']['#description'] = t('Displays the various administrative menu options in the admin bar.');
+  $form['components']['admin_bar.search']['#description'] = t('A drop-down version of the top-level links in the Administration Menu.');
+  $form['components']['admin_bar.users']['#description'] = t('A count of the people currently logged into the site.');
+  $form['components']['admin_bar.account']['#description'] = t(' Links to view and log out of the current user account.');
```

Comment:
Remove the extra space at the beginning of the string.

Reply:
Sorry for that, it didn't render on the website, so I didn't see that.

Reply:
@RakeshPotnuru - It is always good to look at the "Files Changed" tab at the top of this page to review your work. You will notice things here that you might otherwise miss.